.github/workflows/stage-deploy.yml

```diff
@@ -0,0 +1,25 @@
+name: Stage Deploy
+
+on:
+  workflow_dispatch:
+  workflow_call:
+
+env:
+  image-name: ${{ secrets.DOCKER_BASE_REPO_STAGE }}
+  image-tag: ${{ github.event_name == 'workflow_call' && 'latest' || github.sha }}
+
+jobs:
+  push:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Deploy new version 
+        uses: ./.github/actions/deploy
+        with:
+          image-tag: ${{ env.image-tag }}
+          nomad-job-file: anon-da-node-stage.hcl
+          health-checks: http://49.13.145.234:9130/tor/status-vote/current/consensus|http://5.161.108.187:9130/tor/status-vote/current/consensus|http://5.78.90.106:9130/tor/status-vote/current/consensus
```

Comment:
Let's move health check URLs to `env` section, e.g. `da-1-hc-url`, `da-2-hc-url`, `da-3-hc-url`

Then use it like:
```
health-checks: "${{ env.da-1-hc-url }}|${{ env.da-2-hc-url }}|${{ env.da-3-hc-url }}"
```